### PR TITLE
new unit test

### PR DESCRIPTION
**Pull request by anandrkskd**
Signed-off-by: anandrkskd <anandrkskd@gmail.com>

Fixes: #5879

### CODE REVIEW

File: ci-operator/config/redhat-developer/odo/redhat-developer-odo-main.yaml [hunk 1]
```diff
@@ -8,18 +8,27 @@ resources:
...
   steps:
+    pre:
+    - as: install-golint
+      commands: make goget-tools
+      from: src
+      resources:
+        requests:
+          cpu: "2"
+          memory: 4Gi
+      timeout: 4h0m0s
```
Comment (dharmit) [inline ci-operator/config/redhat-developer/odo/redhat-developer-odo-main.yaml:22]: Are we setting 4 hours' timeout for installing golint?

Comment (anandrkskd) [reply ci-operator/config/redhat-developer/odo/redhat-developer-odo-main.yaml:22]: Thanks, @dharmit, that was a copy and paste mistake, I updated the main test timeout but forgot to update this one. Updating it ASAP

**Pull request merged**